quill-core/src/main/scala/io/getquill/context/TranslateContext.scala

```diff
@@ -0,0 +1,43 @@
+package io.getquill.context
```

Comment:
we probably could move this to `Context` directly if we decide not to transform params into strings